src/library_wasmfs_opfs.js

```diff
@@ -363,6 +367,8 @@ addToLibrary({
     wasmfsOPFSProxyFinish(ctx);
   },
 
+  _wasmfs_opfs_write_access__i53abi: true,
+  _wasmfs_opfs_write_access__sig: 'iipij',
```

Comment:
None of these sigs should be needed since I believe they are already stored in src/library_sigs.js

Reply:
The sigs need to be updated.  Not sure if those changes have been added to this request.

Reply:
Yes, changes to function signatures should be accompanied by the change in `library_sigs.js`.   I normally rebuild using `./test/runner other.test_gen_sig_info --rebase`.

Reply:
I've removed the sigs.  Just need to make sure that they are updated.

Reply:
`__sig` attributes are pre-calculated for builtin JS libraries (via `tools/maint/gen_sig_info.py`).    For external libraries you need to specifiy the `__sig` manually, alongside the function.

I made a change recently that will make it impossible to forget to add the `__sig` when using `__i53abi`: #22557